### ISSUE TITLE
Collapse: Remove `toggle` option

### ISSUE THREAD

**Issue by GeoSot** [twbs/bootstrap#34836]
https://github.com/twbs/bootstrap/blob/d9254c64fa10f465a671a47ce40779807f092f5a/js/src/collapse.js#L95-L97

Option toggle can be removed, as we can handle it calling the method `show`

**Comment by patrickhlauke** [comment]
would we not still want to keep the *option*, as the point is that this allows passing on an initial state at instantiation *without* needing to call any explicit method?

**Comment by philipstratford** [comment]
@patrickhlauke It doesn't make sense as an option. The element's initial state can be set with CSS classes (`.collapse` if it's to be hidden, `.collapse.show` if it should be visible initially).

`new bootstrap.Collapse("#myCollapse")` should simply create and return the collapse instance. The trade-off for intuitiveness of having to subsequently call `myCollapse.show()` is more than acceptable, in my opinion.

**Comment by patrickhlauke** [comment]
> @patrickhlauke It doesn't make sense as an option. The element's initial state can be set with CSS classes (`.collapse` if it's to be hidden, `.collapse.show` if it should be visible initially).

but you then also need to make sure to set initial `aria-expanded` state. a lot of "the author will just take care of it themselves"

**Comment by philipstratford** [comment]
> but you then also need to make sure to set initial `aria-expanded` state.

Good point. Could the `new bootstrap.Collapse("#myCollapse")` method return the collapse instance and set the `aria-expanded` attribute's value according to the element's current state? If the element has only the class `.collapse` then add `aria-expanded="false"`. If it has `.collapse.show` then add `aria-expanded="true"`.

**Comment by mdo** [comment]
Fixed in #42799, merged to `v6-dev`. The Collapse `toggle` option is removed. Call `toggle()`, `show()`, or `hide()` directly instead.
